src/command_modules/azure-cli-acr/azure/cli/command_modules/acr/_params.py

```diff
@@ -98,4 +99,20 @@ def load_arguments(self, _):
         c.argument('build_args', nargs='+', help='The space-separated build arguments in a format of <name>=<value>.')
         c.argument('secret_build_args', nargs='+', help='The space-separated secret build arguments in a format of <name>=<value>.')
         c.argument('no_logs', options_list=['--no-logs'], action='store_true', help='Do not show logs after successfully queuing the build.')
+
+    with self.argument_context('acr build-task') as c:
+        c.argument('registry_name', options_list=['--registry', '-r'], help='The name of the container registry. You can configure the default registry name using `az configure --defaults acr=<registry name>`', completer=get_resource_name_completion_list(REGISTRY_RESOURCE_TYPE), configured_default='acr')
+        c.argument('build_task_name', options_list=['--name', '-n'], help='The name of the build task', completer=get_resource_name_completion_list(BUILD_TASK_RESOURCE_TYPE))
+        c.argument('image_name', options_list=['--image', '-t'], help="The image repository and optionally a tag in the 'repository:tag' format.")
+        c.argument('source_location', options_list=['--context'], help="The url to a git repository (eg, 'https://github.com/octocat/hello-worId')")
```

Comment:
It seems redundant to call the parameter `source_location` but also provide an option called `context`. Can you call the parameter `context` at the first place?

Reply:
url -> URL, remove the eg sentence, add the examples in help.